_layouts/post.html

```diff
@@ -3,45 +3,53 @@
 main:
   class: usa-grid usa-section usa-content usa-layout-docs
 ---
-
-<div class="usa-section">
-  <div class="grid-container">
-    <div class="grid-row grid-gap flex-align-start">
-      <article class="usa-layout-docs usa-layout-docs__main usa-prose{% if sidenav %} desktop:grid-col-9{% endif %}">
-        {% if page.title %}
-          <h1>{{ page.title }}</h1>
-        {% endif %}
-            {% if page.author %}
-            {% comment %}Loop through all of the authors of the page.{% endcomment %}
-            {% for author in page.author %}
-            {% comment %}"where" creates an array, so using "first" we can pop the first item off of the array of 1.{% endcomment %}
-            {% assign author_data = site.people | where:"author", author | first %}
-            <div class="author">
-                <p><a href="{{ site.baseurl }}{{ author_data.url }}"><img src="{{ site.baseurl }}/assets/img/people/{{ author_data.image }}" alt="{{ author_data.name }}" title="{{ author_data.name }}"></a> 
-                By <a href="{{ site.baseurl }}{{ author_data.url }}"> {{ author_data.name }}</a> &middot; {{ page.date | date: "%B %-d, %Y"}}</p>
+<main class="" id="main-content">
```

Comment:
@lukefretwell this ``<main>`` is likely going to clash with the ``<main>`` from the default.html file.